nat-lab/tests/test_direct_connection.py

```diff
@@ -861,10 +610,79 @@ async def test_direct_working_paths_with_pausing_upnp_and_stun(
 
         assert len(stun_upnp_requests) == 0
 
-        # LLT-5532: To be cleaned up...
-        alpha_client.allow_errors(
-            ["telio_proxy::proxy.*Unable to send. WG Address not available"]
-        )
-        beta_client.allow_errors(
-            ["telio_proxy::proxy.*Unable to send. WG Address not available"]
+
+UHP_FAILING_PATHS_PARAMS = [
+    [
+        (ConnectionTag.DOCKER_CONE_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_SYMMETRIC_CLIENT_1, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_CONE_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_UDP_BLOCK_CLIENT_1, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_SYMMETRIC_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_SYMMETRIC_CLIENT_2, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_SYMMETRIC_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_UDP_BLOCK_CLIENT_1, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_UDP_BLOCK_CLIENT_1, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_UDP_BLOCK_CLIENT_1, ANY_PROVIDERS, False),
+        (ConnectionTag.DOCKER_UDP_BLOCK_CLIENT_2, ANY_PROVIDERS, False),
+    ],
+    [
+        (ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_1, ["local"], False),
+        (ConnectionTag.DOCKER_FULLCONE_CLIENT_1, ["local"], False),
+    ],
+    [
+        (ConnectionTag.DOCKER_CONE_CLIENT_1, ["local"], False),
+        (ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_1, ["local"], False),
+    ],
+    [
+        (ConnectionTag.DOCKER_SYMMETRIC_CLIENT_1, ["local"], False),
+        (ConnectionTag.DOCKER_OPEN_INTERNET_CLIENT_1, ["local"], False),
+    ],
+]
+
+UHP_FAILING_PATHS = [
+    pytest.param(
+        _generate_setup_parameters(clients),
+    )
+    for clients in UHP_FAILING_PATHS_PARAMS
+]
+
+
+@pytest.mark.asyncio
+@pytest.mark.long
+@pytest.mark.timeout(timeouts.TEST_DIRECT_FAILING_PATHS_TIMEOUT)
+@pytest.mark.parametrize("setup_params", UHP_FAILING_PATHS)
+# Not sure this is needed. It will only be helpful to catch if any
+# libtelio change would make any of these setup work.
+async def test_direct_failing_paths(setup_params: List[SetupParameters]) -> None:
```

Comment:
We could actually get rid of this test in general or atleast add `mark.skip`, this test shouldn't be run in CI

Reply:
It's marked as long, and since we have: `PYTEST_MARKS: "not long"` in our nat-lab job it's skipped anyway